samples/S8.Compose/README.md

```
@@ -0,0 +1,23 @@
+cd samples/S8.Compose
+# Build images and run services
+docke compose up --build
+```
+
+Services:
+- MongoDB: localhost:27017
+- RabbitMQ: AMQP at localhost:5672; Management UI at http://localhost:15672 (guest/guest)
+- S8 Flow API: http://localhost:5108
+  - Health: GET /api/health
+  - Views: GET /views/canonical, /views/lineage (from Sora.Flow.Web)
+  - Static monitor: http://localhost:5108/
```

markdownlint-cli2 (0.17.2)

Bare URL used

(MD034, no-bare-urls)